dialogue-core/src/main/metrics/dialogue-core-metrics.yml

```diff
@@ -2,13 +2,17 @@ options:
   javaPackage: com.palantir.dialogue.core
   javaVisibility: packagePrivate
 namespaces:
-  dialogue.client:
-    docs: Dialogue client response metrics.
+  client:
```

Comment:
linking to c-j-r's metric schema: https://github.com/palantir/conjure-java-runtime/blob/8de52360deb5a05cd9ff68c11e6655767cdd9cb2/okhttp-clients/src/main/metrics/okhttp-metrics.yml#L34-L48

wanna put `deprecations` under here too?

Reply:
Good call!